articles/api-auth/api-access-policies.md

```diff
@@ -0,0 +1,39 @@
+---
+  description: Writing rules to restrict user/client access to an API
+---
+
+# Restrict Client or User Access to APIs
+
+By default, any user associated with an Auth0 client can request an API's scope(s). If you would like to restrict access to the API's scopes based on the user's role, you can do so via [rules](/rules). Then, if a restricted user attempts to request scopes not permitted to those with their role, they will receive an HTTP 401 response.
```

Comment:
` If you would like to restrict access to the API's scopes based on the user's role,`

Not only user roles but any condition we can think of, i.e. we could restrict a certain client id or user id altogether, we could use location, etc